rpcs3/Emu/Cell/PPUTranslator.cpp

```diff
@@ -4909,4 +4911,184 @@ MDNode* PPUTranslator::CheckBranchProbability(u32 bo)
 	return nullptr;
 }
 
+void PPUTranslator::build_interpreter()
+{
+#define BUILD_VEC_INST(i) { \
+		m_function = llvm::cast<llvm::Function>(m_module->getOrInsertFunction("op_" #i, get_type<void>(), m_thread_type->getPointerTo()).getCallee()); \
```

Comment:
This can still be a macro, but to ease on the compiler it can call a lambda inside. (using member function pointer)